src/Symfony/Component/VarDumper/Caster/DateCaster.php

```diff
@@ -68,4 +70,24 @@ private static function formatInterval(\DateInterval $i)
 
         return $i->format(rtrim($format));
     }
+
+    public static function castTimeZone(\DateTimeZone $timeZone, array $a, Stub $stub, $isNested, $filter)
+    {
+        $title = '';
+        if (($location = $timeZone->getLocation()) && method_exists('\Locale', 'getDisplayRegion')) {
+            try {
+                $title = \Locale::getDisplayRegion('-'.$location['country_code'], null);
+            } catch (\Exception $e) {
```

Comment:
I finally handled this with ``extension_loaded('intl')`` but I'm not sure. Is it good for you?